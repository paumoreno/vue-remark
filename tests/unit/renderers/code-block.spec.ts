import VueRemarkCodeBlock from "@/renderers/code-block.vue";
import { mount } from "@vue/test-utils";

describe(VueRemarkCodeBlock.name, () => {
  test("snapshot", () => {
    const wrapper = mount(VueRemarkCodeBlock, {
      propsData: {
        lang: "js",
        value: "import foo from 'bar';"
      }
    });

    expect(wrapper).toMatchSnapshot();
  });
});
