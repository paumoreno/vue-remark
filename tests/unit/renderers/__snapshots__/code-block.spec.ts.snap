// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VueRemarkCodeBlock snapshot 1`] = `
<pre>  <code class="language-js">import foo from 'bar';</code>
</pre>
`;
